IPR Policy.md

```diff
@@ -24,7 +24,7 @@ All [Contributors][contributor] make commitments regarding copyrights; all [Cont
 #### 2.2.2. "Essential Review Draft Claims"
 "Essential Review Draft Claims", with respect to a [Review Draft][review-draft], means all claims in any patent or patent application in any jurisdiction in the world that would necessarily be infringed by implementation of the [Review Draft][review-draft]. A claim is necessarily infringed hereunder only when it is not possible to avoid infringing it because there is no non-infringing alternative for implementation of the [Normative] portions of the [Review Draft][review-draft]. Existence of a non-infringing alternative shall be judged based on the state of the art at the time the [Review Draft][review-draft] is published.
 
-#### 2.2.3. The following are expressly excluded from and shall not be deemed to constitute [Essential Patent Claims]:
+#### 2.2.3. The following are expressly excluded from and shall not be deemed to constitute Essential Patent Claims:
```

Comment:
"Essential Patent Claims" is a defined term, so I think it would be good to link to the definition instead of removing it.. I believe this can be done as #22-essential-patent-claims

Reply:
The issue is actually that your script doesn't seem to process links inside headings, so the brackets pass through unchanged. Elsewhere in the document, `[Essential Patent Claims]` works fine.

We also generally seem to not link to terms inside headings, e.g. https://github.com/whatwg/sg/blob/master/Workstream%20Policy.md has a lot of headings with "Workstream" or "Editors" or "Living Standards". So I thought this was more consistent.

That said, we may want to try to find a more general fix for the Markdown processing mismatch between GitHub and the script, so that we don't have to remember to escape any parenthesis that follow a closing ].

Reply:
Oh, I see. Not linking in headings seems fine. I should probably fix the script to support it correctly, but I probably won't do it right away. In the short term, your way of fixing it seems right.